src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptFindUsagesService.cs

```diff
@@ -22,10 +23,10 @@ internal sealed class VSTypeScriptFindUsagesService(IVSTypeScriptFindUsagesServi
     {
         private readonly IVSTypeScriptFindUsagesService _underlyingService = underlyingService;
 
-        public Task FindReferencesAsync(IFindUsagesContext context, Document document, int position, CancellationToken cancellationToken)
+        public Task FindReferencesAsync(IFindUsagesContext context, Document document, int position, OptionsProvider<ClassificationOptions> classificationOptions, CancellationToken cancellationToken)
```

Comment:
more of question vs a concern.  Why pass in an options provider, vs the options themselves?

Reply:
OptionsProvider is needed when we don't know what language are we going to need options for